examples/slider/js/slider-multithumb.js

```diff
@@ -0,0 +1,345 @@
+        this.minInputNode.max = value;
+        this.maxInputNode.value = value;
+      }
+    }
+  }
+
+  onSliderKeydown(event) {
+    var flag = false;
+    var sliderNode = event.currentTarget;
+    var value = this.getValue(sliderNode);
+    var valueMin = this.getValueMin(sliderNode);
+    var valueMax = this.getValueMax(sliderNode);
+
+    switch (event.key) {
+      case 'ArrowLeft':
+      case 'ArrowDown':
+        this.moveSliderTo(sliderNode, value - 1);
+        flag = true;
+        break;
+
+      case 'ArrowRight':
+      case 'ArrowUp':
+        this.moveSliderTo(sliderNode, value + 1);
+        flag = true;
+        break;
+
+      case 'PageDown':
+        this.moveSliderTo(sliderNode, value - 10);
+        flag = true;
+        break;
+
+      case 'PageUp':
+        this.moveSliderTo(sliderNode, value + 10);
+        flag = true;
+        break;
+
+      case 'Home':
+        this.moveSliderTo(sliderNode, valueMin);
+        flag = true;
+        break;
+
+      case 'End':
+        this.moveSliderTo(sliderNode, valueMax);
+        flag = true;
+        break;
+
+      default:
+        break;
+    }
+
+    if (flag) {
+      event.preventDefault();
+      event.stopPropagation();
+    }
+  }
+
+  onSliderFocus() {
+    this.domNode.classList.add('focus');
+  }
+
+  onSliderBlur() {
+    this.domNode.classList.remove('focus');
+  }
+
+  onSliderMousedown(event) {
+    var onMousemove = function (e) {
+      var diffX = e.pageX - self.railNode.getBoundingClientRect().left;
+
+      if (isMinSlider) {
+        diffX -= self.thumbWidth / 2;
+      } else {
+        diffX -= (3 * self.thumbWidth) / 2;
+      }
+
+      var value = parseInt(
+        ((self.sliderMaxValue - self.sliderMinValue) * diffX) / self.railWidth
+      );
+
+      self.moveSliderTo(sliderNode, value);
+
+      e.preventDefault();
+      e.stopPropagation();
+    };
+
+    var onMouseup = function () {
+      document.removeEventListener('mousemove', onMousemove);
+      document.removeEventListener('mouseup', onMouseup);
+    };
+
+    var sliderNode = event.currentTarget;
+    var isMinSlider = this.isMinSlider(sliderNode);
+    var self = this;
+
+    // bind a mousemove event handler to move pointer
+    document.addEventListener('mousemove', onMousemove);
+
+    // bind a mouseup event handler to stop tracking mouse movements
+    document.addEventListener('mouseup', onMouseup);
+
+    event.preventDefault();
+    event.stopPropagation();
+
+    // Set focus to the clicked handle
+    sliderNode.focus();
+  }
+
+  onInputChange(event) {
+    var tgt = event.currentTarget,
+      value = tgt.value,
+      isNumber = typeof parseInt(value) === 'number';
+
+    //    console.log('\n[onInputChange][   label]: ' + tgt.getAttribute('aria-label'));
+    //    console.log('[onInputChange][isNumber]: ' + isNumber);
+    //    console.log('[onInputChange][   value]: ' + value + ' ' + value.length + ' ' + typeof value);
```

Comment:
Might want to get ride of these :)

Reply:
@spectranaut 
Thanks for pointing this out, I just pushed an update that removed those lines